media/js/firefox/os/firefox-os-2.5.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+;(function($) {
+    'use strict';
+
+    $('.cta-button').on('click', function(e) {
+        e.preventDefault();
+        ftGoalTag14004();
+    });
+
+    function ftGoalTag14004(){
+        var ftRand = Math.random() + "";
+        var num = ftRand * 1000000000000000000;
+        var ftGoalTagPix14004 = new Image();
+        ftGoalTagPix14004.src = "http://servedby.flashtalking.com/spot/2713;14004;1723/?spotName=submit_button& cachebuster="+num;
+        ftGoalTagPix14004.onload = ftLoaded14004;
```

Comment:
This is currently breaking the button for any user with Tracking Protection enabled, since it prevents the image from loading and the onload event doesn't fire. (oh, the irony). I suggest we just use a short delay using `setTimeout` instead.